src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -715,4 +722,53 @@ describe('isAnswering781aQuestions', () => {
       expect(needsToAnswerUnemployability(formData)).to.be.false;
     });
   });
+
+  describe('concatIncidentLocationString', () => {
+    it('should concat full address', () => {
+      const locationString = concatIncidentLocationString({
+        city: 'Test',
+        state: 'TN',
+        country: 'USA',
+        additionalDetails: 'details',
+      });
+
+      expect(locationString).to.eql('Test, TN, USA, details');
+    });
+
+    it('should handle null and undefined values', () => {
+      const locationString = concatIncidentLocationString({
+        city: 'Test',
+        state: null,
+        additionalDetails: 'details',
+      });
+
+      expect(locationString).to.eql('Test, details');
+    });
+  });
+
+  describe('getFlatIncidentKeys', () => {
+    it('should return correct amount of incident keys', () => {
+      expect(getFlatIncidentKeys().length).to.eql(PTSD_INCIDENT_ITERATION * 2);
+    });
+  });
+
+  describe('getPtsdChangeText', () => {
+    it('should have valid labels', () => {
+      Object.keys(PTSD_CHANGE_LABELS).forEach(key => {
+        expect(PTSD_CHANGE_LABELS[key]).to.be.a('string');
```

Comment:
Ahh, this is great! :tada: I think the only scenario we're not covering now is if the key isn't in the constant. That'll still do the wrong thing silently.

I hate to keep asking for more tests, but could you make another one to iterate through all the places you're using this constant and ensure that the all the properties are found in it? I'd hate to get caught on something silly like renaming a property, y'know?